src/library/interface_mono.cpp

```diff
@@ -9,20 +9,46 @@ OrbSlam2InterfaceMono::OrbSlam2InterfaceMono(const ros::NodeHandle& nh,
     : OrbSlam2Interface(nh, nh_private) {
   // Getting data and params
   subscribeToTopics();
-  //advertiseTopics();
-  //getParametersFromRos();
+
+  if(use_body_transform_){
+    if(!getBodyTransform()){
+      ros::shutdown();
+      exit(1);
```

Comment:
I think I see the logic here... However, I would rather check for the body transform when loading parameters from the ROS param server (see below).